src/v2/Apps/Onboarding/config.ts

```diff
@@ -0,0 +1,70 @@
+import { Engine } from "v2/Utils/Engine"
+
+export const engine = new Engine({
+  workflow: [
+    "Welcome",
+    {
+      doYouWantQuestionnaire: {
+        true: [
+          "AsACollector",
+          {
+            whatDoYouLoveMost: {
+              growTasteInArt: [
+                "GrowTasteInArt",
+                {
+                  whereWouldYouLikeToDiveIn: {
+                    auctionHighlights: ["AuctionHighlights"],
+                    trendingArtists: ["TrendingArtists"],
+                    curatedArtworks: ["CuratedArtworks"],
+                  },
+                },
+              ],
+              collectArtThatMovesMe: [
+                "CollectArtThatMovesMe",
+                {
+                  whereWouldYouLikeToDiveIn: {
+                    findNewFromArtistsICollect: ["FindNewFromArtistsICollect"],
+                    trendingArtists: ["TrendingArtists"],
+                    auctionHighlights: ["AuctionHighlights"],
+                  },
+                },
+              ],
+              findNextInvestment: [
+                "FindNextInvestment",
+                {
+                  whereWouldYouLikeToDiveIn: {
+                    keepTrackOfArtistCareer: ["KeepTrackOfArtistCareer"],
+                    trendingArtists: ["TrendingArtists"],
+                    trendingLots: ["TrendingLots"],
+                  },
+                },
+              ],
+              keepTrackOfInterests: [
+                "KeepTrackOfInterests",
+                {
+                  whereWouldYouLikeToDiveIn: {
+                    findNewFromArtistsICollect: ["FindNewFromArtistsICollect"],
+                    followGalleriesILove: ["FollowGalleriesILove"],
+                    createArtWishlist: ["CreateArtWishlist"],
+                  },
+                },
+              ],
+            },
```

Comment:
Note to self for when we pick it back up on Thursday. I think we might be able to flatten this down into two `views` followed by all of the outcome leafs under a single condition. Which should simplify the flow a little bit.